subsys/net/lib/lwm2m/Kconfig

```diff
@@ -235,6 +235,15 @@ config LWM2M_FIRMWARE_UPDATE_OBJ_SUPPORT
 	help
 	  Include support for LWM2M Firmware Update Object (ID 5)
 
+config LWM2M_FIRMWARE_INSTANCE_COUNT
```

Comment:
This could be marked as experimental - please see a [similar](https://github.com/zephyrproject-rtos/zephyr/pull/41158#issuecomment-996483962) suggestion from me for further details.

Reply:
Good suggestion. Unfortunately that doesn't apply to `int` configs, so something different is needed here.